examples/django/django_exampleapp/README.md

```diff
@@ -0,0 +1,36 @@
+# Django Sample Application for Pybrake
+
+## About the application:
+
+The example application provides three GET endpoints:
+
+1. `/date` - returns server date and time. 
+2. `/locations` - returns list of available locations. 
+3. `/weather/{locationName}` - returns the weather details for the locations.
+
+## Steps to run the API:
+
+1. Install the dependencies for the application
+
+```bash
+pip3 install -r requirements.txt
+```
+
+2.  Run the localhost server
+
+```bash
+python manage.py runserver 3000
+```
+
+3. To retrieve the responses, append the endpoints to the localhost URL with a `/`.
+
+Use the below curl commands to interact with the endpoints. The endpoints require an api-key HTTP header.
+
+```bash
+curl "http://localhost:3000/date" -H 'api-key: b761be830f7c23ebe1c3250d42c43673' 
+curl "http://localhost:3000/locations" -H 'api-key: b761be830f7c23ebe1c3250d42c43673' 
+curl "http://localhost:3000/weather/<austin/pune/santabarbara>" -H 'api-key: b761be830f7c23ebe1c3250d42c43673' 
+curl "http://localhost:3000/weather/" -H 'api-key: b761be830f7c23ebe1c3250d42c43673'
```

Comment:
I personally find these `b761be830f7c23ebe1c3250d42c43673` API keys kind of confusing because they look just like Airbrake project API keys. I could imagine a customer getting confused by this. 

I'm not sure a great way around this. Maybe a value like these would be more obviously unrelated to Airbrake credentials:
- `weather-app-key`
- `weather-example-app-key-123`
- `123weatherAppKey123`

Reply:
It's just a hard coded value, and we could change it to something more descriptive like @thompiler suggested.

Reply:
Hey @scottsbaldwin, should this `api-key` being passed in the curl commands for the weather app be authenticated in any way? @salonijain0918 pointed out to me that Chiman's go version of the weather app does have some basic [authentication middleware](https://github.com/airbrake/gobrake/pull/227/files#diff-dc97aeac29b833333a6571446a5b2a501d20d701aad70238f9625b08fc5b3d94R108-R125). The [swagger](https://gist.githubusercontent.com/scottsbaldwin/5f87c2723232f702d5bb434e587c12f2/raw/f6dcbe5208aaf4b230a50445d65faa882ddb326d/weather-api.yaml) for this app though only defines 200 responses (no 401 responses as far as I can see).

Some possible ways the weather app could act:
- Always returns 200 for all curls to valid routes regardless of the value of `api-key` in the header or whether or not it is present.
- Always returns 200s for all curls to valid routes if some `api-key` header is present (ie: it can be any value), otherwise returns 401.
- Only returns 200s for all curls to valid routes if a specific `api-key` header is present (ie: only the api-key in the example would be accepted). (This is what Chiman's example app does), otherwise, return 401.

Reply:
Hey @scottsbaldwin, Django framework has its own in-built middleware for TokenAuthentication, but it requires the use of database to validate the token, I did some research and had a word with @thompiler as well.
and that would also require use of separate route for validation.

Reply:
We can also make use of JWT which would generate a fresh token for each session, please suggest.

Reply:
I've updated the swagger and removed the `api-key` requirement. Let's not do that part.